### ISSUE TITLE
Error: Can't resolve 'es5-ext/to-short-string-representation'

### ISSUE THREAD

**Issue by jeffijoe** [medikoo/memoizee#86]
Memoizee v0.4.8

Occurs in Webpack 3 build.

OS: macOS Sierra

```
Error: Can't resolve 'es5-ext/to-short-string-representation'
```

Latest known working version: 0.4.6.

**Comment by medikoo** [comment]
@jeffijoe thanks. it requires`es5-ext` at at least `0.10.28` which indeed was not forced in dependencies configuration

Fixed with `0.4.9`

**Comment by jeffijoe** [comment]
Excellent, thanks for the quick fix @medikoo !
